### PR TITLE
Implement chain_getMinTransactionFee rpc

### PR DESCRIPTION
**Pull request by HoOngEe**
Implemented `chain_getMinTransactionFee`.
It closes https://github.com/CodeChain-io/codechain/issues/1431

### CODE REVIEW

**Comment by HoOngEe** [comment]
Now the string pattern matched in chain rpc function. I Implemented like this because the only usage of the type indicating string is unique especially in this rpc. But I think it's more appropriate for the matching between these two information to be immersed in `EngineInfo` trait method.
Any Opinion?

File: rpc/src/v1/impls/chain.rs [hunk 1]
```diff
@@ -239,6 +239,29 @@ where
         Ok(self.client.block(&BlockId::Hash(block_hash)).map(|block| block.transactions_count()))
     }
 
+    fn get_min_transaction_fee(&self, action_type: String) -> Result<Option<u64>> {
+        let common_parameters = self.client.common_params();
+        Ok(match action_type.as_str() {
+            "MintAsset" => Some(common_parameters.min_asset_mint_cost),
```
Comment (sgkim126) [inline rpc/src/v1/impls/chain.rs]: I think it's better to use camelCase.
Because the action type of RPC uses camelCase.

**Comment by sgkim126** [comment]
I think a string match in the RPC method is a fine option because it's only used here.

File: rpc/src/v1/impls/chain.rs (hunk 1 above)
Comment (HoOngEe) [reply rpc/src/v1/impls/chain.rs]: I changed them to `camelCase`s

File: rpc/src/v1/traits/chain.rs [hunk 2]
```diff
@@ -124,6 +124,10 @@ build_rpc_trait! {
          # [rpc(name = "chain_getBlockTransactionCountByHash")]
         fn get_block_transaction_count_by_hash(&self, H256) -> Result<Option<usize>>;
 
+        ///Gets the minimum transaction fee of the given name.
+         # [rpc(name = "chain_getMinTransactionFee")]
+        fn get_min_transaction_fee(&self, String) -> Result<Option<u64>>;
```
Comment (sgkim126) [inline rpc/src/v1/traits/chain.rs]: You need a block number.

Comment (HoOngEe) [reply rpc/src/v1/traits/chain.rs]: I added the block_number parameter.

**Pull request merged**